### PR TITLE
fix: show location for ICE

### PR DESCRIPTION
**Pull request by JonathanStarup**

### CODE REVIEW

**Comment by JonathanStarup** [comment]
Related discussion #8541

File: main/src/ca/uwaterloo/flix/util/InternalCompilerException.scala [hunk 1]
```diff
@@ -25,4 +25,4 @@ import ca.uwaterloo.flix.language.ast.SourceLocation
   *
   * @param message the error message.
   */
-case class InternalCompilerException(message: String, loc: SourceLocation) extends RuntimeException(message)
+case class InternalCompilerException(message: String, loc: SourceLocation) extends RuntimeException(s"$message ($loc)")
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/util/InternalCompilerException.scala:28]: Do you need to call .format?

Comment (JonathanStarup) [reply main/src/ca/uwaterloo/flix/util/InternalCompilerException.scala:28]: toString calls format

**Review by magnus-madsen — APPROVED** [review]

**Comment by jaschdoc** [comment]
Nice

**Pull request merged**